tests/robustness/multiclient_test/multiclient_test.go

```diff
@@ -176,3 +176,11 @@ func tryRandomAction(ctx context.Context, t *testing.T, opts engine.ActionOpts)
 
 	return err
 }
+
+func assertNoError(t *testing.T, err error) {
+	t.Helper()
+
+	if err != nil {
+		t.Errorf("err: %v", err)
```

Comment:
`assert` often implies the check should be fatal, especially since other outputs are often nil when a non-nil error is returned; this can lead to nil-pointer deref if execution continues. This should probably be changed to 
```suggestion
		t.Fatalf("err: %v", err)
```

Reply:
Thanks for catching this. I added this to catch up with some upstream Kopia changes (see https://github.com/kopia/kopia/pull/1058) but I think based on the intent of that change, I should be using `require.NoError(t, err)` instead of implementing this `assertNoError` function. I'll fix it.